main.js

```diff
@@ -23,11 +23,32 @@
 
 
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
-/*global define, $ */
+/*global define */
 
-define(function (require, exports, module) {
+define(function (require, exports) {
     "use strict";
-    
+
+    /**
+     * This helper function will convert a generator domain name into
+     * the key that is used to store its custom preferences in photoshop.
+     *
+     * This assumes that the plugin registered a domain
+     * with the same name as the plugin itself
+     *
+     * The replacement logic must match the way generator-core
+     * See: generator's escapePluginId() function
+     *
+     * @type {?string}
```

Comment:
And `@return`? Or is this a thing where you're matching some other old syntax found in this repo?